servicebuilder-activemq/src/test/java/no/obos/util/servicebuilder/mq/ActiveMqListenerLoggingTest.java

```diff
@@ -0,0 +1,106 @@
+package no.obos.util.servicebuilder.mq;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ActiveMqListenerLoggingTest {
+
+    @Test
+    public void handleNull() {
+        assertThat(ActiveMqListener.truncateMessageForLogging(null)).isEqualTo(null);
+    }
+
+    @Test
+    public void logNormalLengthMessage() {
+        String msg = "{\"mottakerId\":13136040,\"message\":\"Hei Adam! Eva ber om å bli lagt til som beboer"
+                + " i Lade Alle Hjem Boligsameie. Gå til Min bolig på https://vibbo.no/xxxx for å godta forespørselen.\",\"from\":\"Vibbo\",\"to\":\"99999999\"}";
+
+        assertThat(ActiveMqListener.truncateMessageForLogging(msg)).isEqualTo(msg);
+    }
+    @Test
+    public void logLongMessageWithDataObject() {
```

Comment:
Veldig bra med test! 💪🏿 

Jeg får ikke kjørt den, men det er sikkert fordi jeg ikke har satt opp IntelliJ riktig. Vet ikke om jeg gidder gjøre det 😅 
> java: lambda expressions are not supported in -source 1.5
  (use -source 8 or higher to enable lambda expressions)